Charts file added

lakshmipriyasundarraj opened the pull request.

lakshmipriyasundarraj committed:
Folder added

lakshmipriyasundarraj committed:
file added

lakshmipriyasundarraj committed:
axis file

lakshmipriyasundarraj committed:
file imports path changed

akileswari reviewed (commented):
- Don't use d3 for yaxis rendering, remove that and use svg line and text
- Dont create separate label file each chart of x-axis label.
- Remove d3 from axis, chart label

lakshmipriyasundarraj committed:
Worked PR commands

lakshmipriyasundarraj committed:
Complete barchart

lakshmipriyasundarraj committed:
Fixed Waterfall chart

This commit fixes the Waterfall Chart. Reused files are working correctly.

lakshmipriyasundarraj committed:
Font style

In this commit, I created the theme file where I provided styles for all fonts.

lakshmipriyasundarraj committed:
Barchart dynamic

lakshmipriyasundarraj committed:
Before merging

lakshmipriyasundarraj committed:
after merging

lakshmipriyasundarraj committed:
Template issue

lakshmipriyasundarraj committed:
axis integration

lakshmipriyasundarraj committed:
toggle complete

lakshmipriyasundarraj committed:
toolbar icon added

lakshmipriyasundarraj committed:
layout icons added

lakshmipriyasundarraj committed:
dark theme update

lakshmipriyasundarraj committed:
theme completed waterfall

lakshmipriyasundarraj committed:
layout

lakshmipriyasundarraj committed:
layout header

lakshmipriyasundarraj committed:
custom icon

lakshmipriyasundarraj committed:
fixing tool bar UI

lakshmipriyasundarraj committed:
inforiver chart themes

lakshmipriyasundarraj committed:
Resolved merge conflicts

lakshmipriyasundarraj committed:
theme property

lakshmipriyasundarraj committed:
end

lakshmipriyasundarraj committed:
Theme intagration Xaxis Yaxis and DataValues

lakshmipriyasundarraj committed:
Updated theme Xaxis and Yaxis and Datavalues

lakshmipriyasundarraj committed:
Chart theme intagration

lakshmipriyasundarraj committed:
Ui part completed

lakshmipriyasundarraj committed:
end of the day commit

lakshmipriyasundarraj committed:
Layout with function

akileswari commented on src/TemplateBuilder/TemplateWrapper.tsx:
```diff
@@ -1,30 +1,18 @@
-      <TemplateToolbar />
-      <TemplatePreview height={fHeight} width={fWidth} />
+    <div className="template-wrapper" style={{ height: `${divHeight}px`, width: `${divWidth}px` }}>
+      {/* <TemplateToolbar /> */}
```
should not remove the main content

akileswari commented on src/components/Theme/Theme.ts:
```diff
@@ -0,0 +1,26 @@
+  const [currentTheme, setCurrentTheme] = useState('light');
+
+  const toggleTheme = () => {
+    setCurrentTheme((prevTheme) => (prevTheme === 'light' ? 'dark' : 'light'));
```
can assign direct key to state

akileswari commented on src/components/Theme/Theme.ts:
```diff
@@ -0,0 +1,26 @@
+    setCurrentTheme((prevTheme) => (prevTheme === 'light' ? 'dark' : 'light'));
+  };
+
+  const getTheme = (theme: string) => {
```
add return type as theme object

akileswari commented on src/components/Theme/darkTheme.ts:
```diff
@@ -0,0 +1,382 @@
+import themes from "./Theme";
```
Unnecessary import

akileswari commented on src/components/Theme/lightTheme.ts:
```diff
@@ -0,0 +1,385 @@
+import '../Theme/Theme'
```
Unnecessary import

akileswari commented on src/components/Theme/darkTheme.ts:
```diff
@@ -0,0 +1,382 @@
+    ...highLightBoxplotColors,
+};
+
+const darkTheme: IThemeColor = {
```
add IThemeColor interface

akileswari commented on src/components/layout/Layout.css:
remove this file

akileswari commented on src/components/layout/customIcon.css:
move this to css folder

akileswari commented on src/constant/Helper.tsx:
remove these constant , use dynamic dimensions

akileswari reviewed (changes requested):
move all the css file to css folder with component as sub folder

lakshmipriyasundarraj committed:
Layout UI functionality

lakshmipriyasundarraj committed:
PR changes

lakshmipriyasundarraj committed:
Layout Title

akileswari commented on src/ChartComponents/inforiver-dashboard.code-workspace:
remove this file

akileswari commented on src/TemplateBuilder/TemplatePreview.tsx:
```diff
@@ -1,83 +1,195 @@
+        
+        <g transform={`translate(${svgWidth - 100}, 20)`}>
+          <rect width="90" height="30" fill={theme.footer.label} rx="5" />
+          <foreignObject x="0" y="0" width="90" height="30">
```
remove this theme button

akileswari commented on src/components/Axis/dashboard.code-workspace:
remove this file

akileswari commented on src/components/layout-icon/template bilder icon march 8/icons.svg:
remove this file its is not need

akileswari commented on src/components/layout-icon/template bilder icon march 8/css/style.css:
```diff
@@ -0,0 +1,167 @@
+Generated using nucleoapp.com
+
+-------------------------------- */
+.template-builder {
```
move this file to css folder.
add bg image for body instead template-builder

akileswari commented on src/components/layout/layoutGrid.tsx:
```diff
@@ -0,0 +1,21 @@
+const LayoutGrid = () => {
+  return(
+   <div className="grid-container">
+ <div className="rectangle"></div>
```
need to change with react-grid

lakshmipriyasundarraj committed:
PR commands

lakshmipriyasundarraj committed:
Grid layout library

lakshmipriyasundarraj committed:
Layout grid

lakshmipriyasundarraj committed:
current code after merging

akileswari commented on src/ChartComponents/inforiver-dashboard.code-workspace:
remove this file

lakshmipriyasundarraj committed:
layout css

lakshmipriyasundarraj committed:
tool bar UI changes

akileswari committed:
Delete src/ChartComponents/inforiver-dashboard.code-workspace
